.travis.yml

```diff
@@ -91,160 +53,150 @@ matrix:
     fast_finish: true
 
     include:
-        # Try MacOS X.
+
+      # Try MacOS X.
         - os: osx
           stage: Cron tests
-          env: SETUP_CMD='test --remote-data=astropy'
-               CONDA_DEPENDENCIES="$CONDA_ALL_DEPENDENCIES clang"
-               PIP_DEPENDENCIES="scikit-image jplephem $ASDF_PIP_DEP"
+          env: PYTHON_VERSION=3.7
+               TOXENV="test-alldeps"
+               TOXPOSARGS="--remote-data=astropy"
+               CONDA_DEPENDENCIES="clang"
                CCOMPILER=clang
+               USE_CI_HELPERS=True
                EVENT_TYPE='cron'
-               DOUBLE_PLAY=True
+               # DOUBLE_PLAY=True
 
         # Try all python versions and Numpy versions. Since we can assume that
         # the Numpy developers have taken care of testing Numpy with different
         # versions of Python, we can vary Python and Numpy versions at the same
         # time.
-        - os: linux
-          env: PYTHON_VERSION=3.6 NUMPY_VERSION=1.16
-               INSTALL_CMD='python setup.py build_ext --inplace'
-               PIP_DEPENDENCIES='pytest-astropy'
-               TEST_CMD='pytest --open-files --doctest-rst'
-          script:
-            - $INSTALL_CMD
-            - $TEST_CMD
-
-        # Now try with all optional dependencies. We also include the --readonly
-        # flag to make sure no files are being written to the temporary install
-        # location during testing. We also use this build to make sure that the
-        # dependencies get correctly installed with pip.
-        - os: linux
-          env: SETUP_CMD='test --remote-data=astropy --readonly'
-               LC_CTYPE=C.ascii LC_ALL=C
-               PIP_DEPENDENCIES="" CONDA_DEPENDENCIES=""
-               INSTALL_WITH_PIP=True
-               EXTRAS_INSTALL="test,all"
 
-        - os: linux
+        # For the Numpy 1.16 build we use oldestdeps not numpy116 since we can check
+        # the oldest version of all dependencies where this is known.
+        - language: python
+          python: 3.6
+          env: TOXENV="test-oldestdeps"
+               TOXPOSARGS="--open-files"
```

Comment:
Shouldn't testing for open files be in the default?

Reply:
This slows down the running of tests, so we only included it in one build. I'm trying to avoid any actual changes to the test matrix here, focusing on just translating the old one into the new syntax. I suggest we keep any actual configuration changes for a separate PR.

Reply:
Sounds good.  If you have the energy add something like "we also run the (slow) tests for open files here"

Reply:
Done